src/users/containers/users/users.component.ts

```diff
@@ -45,9 +45,9 @@ export class UsersComponent implements OnInit, OnDestroy {
   }
 
   public getAllUsers(): Subscription {
-    return this.usersService.getAllUsersList().subscribe(((allUserList) => {
+    return this.usersService.getAllUsersList().subscribe((allUserList) => {
```

Comment:
2 API calls to the same route here. One to get a subset of data for the table, second to get all the data to get the total count. Possible tech debt to review?